platform/mellanox/mft-fwtrace-cfg/Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+# Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = mft-fwtrace-cfg-$(MFT_VERSION)-$(MFT_REVISION).deb
```

Comment:
@vadymhlushko-mlnx please avoid copy-paste code. Use `MFT_FWTRACE_CFG` from `platform/mellanox/mft-fwtrace-cfg.mk` instead. For that you will need to export the vars in original `.mk`